tests/utilities.py

```
@@ -18,7 +18,7 @@ def stub_async_write_ha_state(entity: Any) -> None:
     object.__setattr__(entity, "async_write_ha_state", lambda: None)
 
 
-def patch_client_factory(monkeypatch: pytest.MonkeyPatch, module: Any, client_ctor: Any) -> None:
+def patch_client_factory(monkeypatch: pytest.MonkeyPatch, module: Any, client_ctor: Any) -> Any:
```

Ruff (0.15.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `module`

(ANN401)